code/modules/admin/verbs/mapping.dm

```diff
@@ -126,9 +122,8 @@ var/intercom_range_display_status = 0
 					qdel(F)
 	feedback_add_details("admin_verb","mIRD") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
-var/list/debug_verbs = list (
-	/client/proc/do_not_use_these
```

Comment:
That exists so it´s also show in in the game that they should not be used in the game.